tap_stripe/__init__.py

```diff
@@ -125,6 +125,7 @@
 LOGGER = singer.get_logger()
 
 DEFAULT_DATE_WINDOW_SIZE = 30 #days
+DEFAULT_EVENT_DATE_WINDOW_SIZE = 7 #days
```

Comment:
```suggestion
DEFAULT_EVENT_UPDATE_DATE_WINDOW = 7 #days
```

Reply:
Added comment to explain `DEFAULT_EVENT_UPDATE_DATE_WINDOW`